plugins/woocommerce-admin/client/products/components/product-link-field.tsx

```diff
@@ -64,24 +65,40 @@ export const ProductLinkField = () => {
 	return (
 		<>
 			<span className="woocommerce-product-form__secondary-text product-details-section__product-link">
-				{ __( 'Product link', 'woocommerce' ) }
-				:&nbsp;
-				<a href={ values.permalink } target="_blank" rel="noreferrer">
-					{ permalinkPrefix }
-					{ values.slug || cleanForSlug( values.name ) }
-					{ permalinkSuffix }
-				</a>
-				<Button
-					variant="link"
-					onClick={ () => setIsModalVisible( true ) }
-				>
-					{ __( 'Edit', 'woocommerce' ) }
-				</Button>
+				{ interpolateComponents( {
+					/* eslint-disable-next-line max-len */
+					/* translators: Sentence fragment describing a product attribute match. Example: "Color Is Not Blue" - attribute = Color, equals = Is Not, value = Blue */
```

Comment:
I don't understand this translator comment... is this perhaps a copy-paste from somewhere?

Reply:
Definitely ©️ 🍝 

Removed; I think the variable naming is adequate similar to our other use cases with `interpolateComponents`.